sw/otbn/code-snippets/p256_isoncurve_test.s

```diff
@@ -0,0 +1,81 @@
+/* Copyright lowRISC contributors. */
+/* Licensed under the Apache License, Version 2.0, see LICENSE for details. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/**
+ *   Standalone test for P-256 curve point test
```

Comment:
Nit: reduce indentation within the block to a single space:

"  *   Standalone test for P-256 curve point test" -> "  * Standalone test for P-256 curve point test"